src/static/css/openforms.css

```diff
@@ -778,7 +825,7 @@ thead {
     display: table-header-group;
     vertical-align: middle;
     border-color: inherit;
-    background-color: var(--ripple-light-neutral);
+    background-color: #ffffff00 !important;
```

Comment:
Can you please stick to using the theme variables, if the colour is only slightly off its better to stick to one we already have setup. Also this hex code is longer than 6 characters